esp/logging/loggingagent/espserverloggingagent/loggingagent.cpp

```diff
@@ -154,6 +160,15 @@ bool CESPServerLoggingAgent::readLogFilters(IPropertyTree* cfg, unsigned groupID
     return hasFilter;
 }
 
+void CESPServerLoggingAgent::createLocalTransactionSeed(StringBuffer& transactionSeed)
+{
+    unsigned ip = queryHostIP().iphash();
```

Comment:
I wonder if it would be worthwhile to generate this once at load-time, and re-use...
the danger would be that the ip could change?